winconf.pri

```diff
@@ -35,12 +32,15 @@ win32-g++* {
     DEFINES += _FILE_OFFSET_BITS=64
     DEFINES += __USE_W32_SOCKETS
 
+    QMAKE_CXXFLAGS += -std=c++17
```

Comment:
same here

Reply:
Now that I think about it, `CONFIG += c++17` should be set once and outside the compiler specific sections.

Reply:
nevermind